libbeat/processors/registry.go

```diff
@@ -1,10 +1,39 @@
 package processors
 
 import (
+	"errors"
+
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
+	p "github.com/elastic/beats/libbeat/plugin"
 )
 
+type processorPlugin struct {
+	name   string
+	constr Constructor
+}
+
+var pluginKey = "libbeat-processor"
```

Comment:
should we use the dot notation for the keys as we do in most other places? Means `libbeat.processor`